InnerEye/ML/metrics.py

```diff
@@ -32,6 +36,140 @@
 from InnerEye.ML.utils.ml_util import check_size_matches
 from InnerEye.ML.utils.sequence_utils import get_masked_model_outputs_and_labels
 
+from pytorch_lightning import metrics
+
+class MeanAbsoluteError(metrics.MeanAbsoluteError):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = MetricType.MEAN_ABSOLUTE_ERROR.value
+
+class MeanSquaredError(metrics.MeanSquaredError):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = MetricType.MEAN_SQUARED_ERROR.value
+
+class ExplainedVariance(metrics.ExplainedVariance):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = MetricType.EXPLAINED_VAR.value
+
+
+class Accuracy05(metrics.Accuracy):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = MetricType.ACCURACY_AT_THRESHOLD_05.value
+
+class ScalarMetricsBase(Metric):
+    def __init__(self, dist_sync_on_step: bool = False, name: str = "metric"):
+        super().__init__(dist_sync_on_step=dist_sync_on_step)
+        self.add_state("preds", default=[], dist_reduce_fx=None)
```

Comment:
preds -> predictions? Or use a name that is consistent with the names that PL metrics classes use

Reply:
They actually use this "preds" and "target" that's why i chose these names